aten/src/ATen/native/native_functions.yaml

```diff
@@ -46,7 +46,7 @@
   variants: function
 
 # Computes the gradient of current tensor w.r.t. graph leaves.
-- func: backward(Tensor self, Tensor? gradient=None, bool? retain_graph=None, bool create_graph=False) -> ()
+- func: backward(Tensor self, Tensor? gradient=None, bool? retain_graph=None, bool create_graph=False, Tensor? inputs=None) -> ()
```

Comment:
You can use `Tensor[]` to get a list of Tensors as argument here!

Reply:
I get a `RuntimeError: type 'c10::optional<TensorList>' is not supported` when trying to change it to `Tensor[]?`

Reply:
Shouldn't `Tensor?[]` work? It is used by `index.Tensor` below so it must be supported.

Reply:
Hmm, doesn't work either. I'm not sure how codegen works yet but maybe it is because Tensor?[] is only special cased [here](https://github.com/pytorch/pytorch/blob/8c39f198b48b278c5298f6c3831ddcae4bf35147/tools/autograd/gen_variable_type.py#L519), but if we check [here](https://github.com/pytorch/pytorch/blob/8c39f198b48b278c5298f6c3831ddcae4bf35147/tools/autograd/gen_variable_type.py#L735), we see that we have marked `backward` to be manually traced